### PR TITLE
[miio] enable telnet for lumi gateway

### PR DESCRIPTION
**Pull request by marcelrv**
* allow enable telnet and some other functions on gateway
* use new fw as default for chugmi plug

Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: bundles/org.openhab.binding.miio/README.md [hunk 1]
```diff
@@ -2806,6 +2847,58 @@ Number SpeedLevel "Fan-Speed Level" (G_fan) {channel="miio:basic:fan:SpeedLevel"
...
+
+note: Autogenerated example. Replace the id (gateway) in the channel with your own. Replace `basic` with `generic` in the thing UID depending on how your thing was discovered.
+
+```java
+Group G_gateway "Xiaomi Mi Mijia Gateway V3 ZNDMWG03LM" <status>
+Switch telnetEnable "Enable Telnet" (G_gateway) {channel="miio:basic:gateway:telnetEnable"}
+Number doorbelVol "Doorbell Volume" (G_gateway) {channel="miio:basic:gateway:doorbelVol"}
+Number gatewayVol "Gateway Volume" (G_gateway) {channel="miio:basic:gateway:gatewayVol"}
+Number alarmingVol "Alarming Volume" (G_gateway) {channel="miio:basic:gateway:alarmingVol"}
+String doorbelPush "Doorbell Push" (G_gateway) {channel="miio:basic:gateway:doorbelPush"}
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/README.md:2899]: You should also fix this spelling mistake everywhere else as well.
```suggestion
String doorbellPush "Doorbell Push" (G_gateway) {channel="miio:basic:gateway:doorbellPush"}
```

**Review by cpmeister — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.miio/README.md (hunk 1 above)
Comment (marcelrv) [reply bundles/org.openhab.binding.miio/README.md]: thanks. fixed.

File: bundles/org.openhab.binding.miio/README.md [hunk 2]
```diff
@@ -1067,6 +1068,46 @@ e.g. `smarthome:send actionCommand 'upd_timer["1498595904821", "on"]'` would ena
...
+| gatewayVol       | Number  | Gateway Volume                      |
+| alarmingVol      | Number  | Alarming Volume                     |
+| doorbellPush     | String  | Doorbell Push                       |
+
+### Mi Smart Home Gateway v2 (<a name="lumi-gateway-v2">lumi.gateway.v2</a>) Channels
+
+| Channel          | Type    | Description                         |
+|------------------|---------|-------------------------------------|
+| telnetEnable     | Switch  | Enable Telnet                       |
+| doorbelVol       | Number  | Doorbell Volume                     |
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/README.md:1086]: Same type here and elsewhere too.
```suggestion
| doorbellVol       | Number  | Doorbell Volume                     |
```

**Review by cpmeister — CHANGES_REQUESTED** [review]

**Comment by marcelrv** [comment]
that was a good reminder again why I should not push a commit when in hurry, my son  literally jumping on my lap and computer etc... :(

**Review by cpmeister — APPROVED** [review]
LGTM

**Pull request merged**